syft/execution/action.py

```diff
@@ -259,15 +257,15 @@ def unbufferize(worker: AbstractWorker, protobuf_obj):
         for key in protobuf_obj.kwargs:
             kwargs_[key] = sy.serde.protobuf.serde.unbufferize_arg(worker, protobuf_obj.kwargs[key])
 
-        return_ids = tuple(
-            [sy.serde.protobuf.proto.get_protobuf_id(pb_id) for pb_id in protobuf_obj.return_ids]
-        )
+        return_ids = [
+            sy.serde.protobuf.proto.get_protobuf_id(pb_id) for pb_id in protobuf_obj.return_ids
+        ]
```

Comment:
This changes the type of the variable, right?

Reply:
yeah...